resources/views/community/slack.blade.php

```diff
@@ -0,0 +1,61 @@
+{{--
+    Copyright 2015 ppy Pty. Ltd.
+
+    This file is part of osu!web. osu!web is distributed with the hope of
+    attracting more community contributions to the core ecosystem of osu!.
+
+    osu!web is free software: you can redistribute it and/or modify
+    it under the terms of the Affero GNU General Public License version 3
+    as published by the Free Software Foundation.
+
+    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+    See the GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+--}}
+@extends("master")
+
+@section("content")
+    <div class="osu-layout__row osu-layout__row--page-compact">
+        <div class="slack-header">
+            <div class="slack-header__text-container">
+                <h2 class="slack-header__text slack-header__text--small">{{ trans('community.slack.header.small') }}</h2>
+                <h1 class="slack-header__text slack-header__text--large">{{ trans('community.slack.header.large') }}</h1>
+            </div>
+        </div>
+    </div>
+
+    <div class="osu-layout__row osu-layout__row--page-compact">
+        <div class="osu-layout__row slack-content">
+            <div class="slack-content__item slack-content__image-container"><img class="slack-content__image" src="/images/layout/slack/slack-logo.png"/></div>
```

Comment:
probably should be `slack-content__item slack-content__item--image` (and `--text` for the one below)